draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -994,21 +994,35 @@ the SD-JWT.
 
 ## Selectively-Disclosable Validity Claims {#sd-validity-claims}
 
-Claims controlling the validity of the SD-JWT, such as `nbf`, `iat`, and `exp`,
-are usually included in plaintext in the SD-JWT payload, but MAY be
-selectively disclosable instead. In this case, however, it is up to the Holder
-to release the claims to the Verifier. A malicious Holder may try to hide, for
+An Issuer MUST NOT allow any content to be selectively disclosable that is critical for evaluating the
+SD-JWT's authenticity or validity.
+The exact list of such content will depend on the application
+and SHOULD be listed by any application-specific profile of SD-JWT.
+The following is a list of typical JWT claim names that SHOULD be considered as
+security-critical:
+
+* `iss` (Issuer)
+* `aud` (Audience), although issuers MAY allow individual entries in the array to be selectively disclosable
+* `exp` (Expiration Time)
+* `nbf` (Not Before)
+* `cnf` (Confirmation Key)
+
+Consequently, claims controlling the validity of the SD-JWT
+will typically be included in plaintext in the SD-JWT payload. Verifiers, however, cannot
+reliably depend on that and need to operate as though security-critical claims might be
+selectively disclosable. In such a case, the onus is on the Holder
+to release the relevant claims to the Verifier. And a malicious Holder may try to hide, for
 example, an expiration time (`exp`) in order to get a Verifier that "fails open"
 to accept an expired SD-JWT.
```

Comment:
sorry, I don't fully understand the purpose of this paragraph.. 
if issuer will typically put in the plaintext, why verifier cannot rely on that?
If there is a possibility of a malicious holder, shouldn't the sentence be phrased more like a security consideration for the verifiers not to "fail open"..?

```suggestion
Consequently, Issuer will typically include claims controlling the validity of the SD-JWT
in plaintext in the SD-JWT payload. Verifiers, however, cannot
reliably depend on that and need to operate as though security-critical claims might be
selectively disclosable. In such a case, the onus is on the Holder
to release the relevant claims to the Verifier. And a malicious Holder may try to hide, for
example, an expiration time (`exp`) in order to get a Verifier that "fails open"
to accept an expired SD-JWT.
```

Reply:
Your suggested text reads much better and I think we can do without the malicious holder example now.